src/modality-specific-files/task-events.md

```diff
@@ -268,7 +268,11 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
         "SoftwareName": "Psychtoolbox",
         "SoftwareRRID": "SCR_002881",
         "SoftwareVersion": "3.0.14",
-        "Code": "doi:10.5281/zenodo.3361717"
+        "Code": "doi:10.5281/zenodo.3361717",
+        "ScreenDistance": 180,
```

Comment:
@Remi-Gau - even though unit was adjusted to become `m`, examples seems were not like here  and below!... I will send a PR

Reply:
Oops. Thanks for spotting it.